app/src/main/res/values/strings.xml

```diff
@@ -800,6 +800,8 @@ You only need to do this once, until the next time you select a new location for
     <string name="error_cannot_get_package_info">Unable to get package info from file \"%s\". Either the specified file is not an APK, or the package file is corrupt.</string>
     <string name="preference_delete_confirmation">Delete confirmation</string>
     <string name="preference_delete_confirmation_summary">Ask for confirmation before deleting files. Disabling this is highly discouraged!</string>
+    <string name="preference_delete_permanently_without_confirmation">Delete permanently</string>
```

Comment:
please change this to "delete permanently (disable trashbin)"

Reply:
Thank you Vishnu, will change it.

Reply:
Please let me know if these strings are okay. I will commit it.

![image](https://github.com/user-attachments/assets/6ec08f4d-cd8d-4d70-811e-79de2b904516)

Reply:
Sure, please do!

Reply:
Done